src/scry/analyzer.cr

```diff
@@ -5,12 +5,22 @@ require "./publish_diagnostic"
 
 module Scry
   struct Analyzer
+    @@source = [] of String
```

Comment:
I've been out of the Scry codebase for a while, but I'm a bit concerned about this being set as a class var. Doesn't this make it impossible to check a project with multiple files against a single running server instance?